core/schema/modulesource.go

```diff
@@ -728,7 +734,7 @@ func (s *moduleSourceSchema) directoryAsModule(
 			},
 		},
 		dagql.Selector{
-			Field: "asModule",
+			Field: "(asModule)",
```

Comment:
wat

Reply:
What the... No idea how this happened 🤷‍♂️